src/styles/shadows.css

```
@@ -0,0 +1,6 @@
+:root {
+  --elevation-0: 0 0 3px rgba(0 0 0 / 24%);
```

HEX тоже позволяет задействовать альфа-канал. Можно использовать его для единообразия
```suggestion
  --elevation-0: 0 0 3px #00000024;
```

---

Done